src/providers/export/tensorFlowRecords/tensorFlowBuilder.ts

```diff
@@ -0,0 +1,148 @@
+     * @description - Add an Int64 feature
+     */
+    public addIntFeature(key: string, value: number) {
+        this.addIntArrayFeature(key, [value]);
+    }
+
+    /**
+     * @key - Feature Key
+     * @value - A Float value
+     * @description - Add a Float feature
+     */
+    public addFloatFeature(key: string, value: number) {
+        this.addFloatArrayFeature(key, [value]);
+    }
+
+    /**
+     * @key - Feature Key
+     * @value - A String value
+     * @description - Add a String feature
+     */
+    public addStringFeature(key: string, value: string) {
+        this.addBinaryArrayFeature(key, [textEncode(value)]);
+    }
+
+    /**
+     * @key - Feature Key
+     * @value - A Uint8Array value
+     * @description - Add an Uint8Array byte array feature
+     */
+    public addBinaryFeature(key: string, value: Uint8Array) {
+        this.addBinaryArrayFeature(key, [value]);
+    }
+
+    /**
+     * @key - Feature Key
+     * @value - An Array of Int64 values
+     * @description - Add an Array of Int64 values feature
+     */
+    public addIntArrayFeature(key: string, values: number[]) {
+        const intList = new Int64List();
+        values.forEach((value) => {
+            intList.addValue(value);
+        });
+
+        const feature = new Feature();
+        feature.setInt64List(intList);
+
+        const featuresMap = this.features.getFeatureMap();
+        featuresMap.set(key, feature);
+    }
+
+    /**
+     * @key - Feature Key
+     * @value - An Array of Float values
+     * @description - Add an Array of Float values feature
+     */
+    public addFloatArrayFeature(key: string, values: number[]) {
+        const floatList = new FloatList();
+        values.forEach((value) => {
+            floatList.addValue(value);
+        });
+
+        const feature = new Feature();
+        feature.setFloatList(floatList);
+
+        const featuresMap = this.features.getFeatureMap();
+        featuresMap.set(key, feature);
+    }
+
+    /**
+     * @key - Feature Key
+     * @value - An Array of String values
+     * @description - Add an Array of String values feature
+     */
+    public addStringArrayFeature(key: string, values: string[]) {
+        this.addBinaryArrayFeature(key, values.map((value) => textEncode(value)));
+    }
+
+    /**
+     * @key - Feature Key
+     * @value - An Array of Uint8Array values
+     * @description - Add an Array of Uint8Array byte array values feature
+     */
+    public addBinaryArrayFeature(key: string, values: Uint8Array[]) {
+        const byteList = new BytesList();
+        values.forEach((value) => {
+            byteList.addValue(value);
+        });
+
+        const feature = new Feature();
+        feature.setBytesList(byteList);
+
+        const featuresMap = this.features.getFeatureMap();
+        featuresMap.set(key, feature);
+    }
+
+    /**
+     * @description - Return a Buffer representation of a single TFRecord
+     */
+    public releaseTFRecord(): Buffer {
```

Comment:
`releaseTFRecord` is a bit confusing as a name.  What about we rename this to `build()` ?